### PR TITLE
Restructuring and repairing the warehouse tutorial

### PR DESCRIPTION
**Pull request by charlie-costanzo**
Finally getting around to:

- fixing the markdown issues from the migration
- making it more readable
- making it more accessible to a new user
- incorporating feedback from current analysts
- Added introductory text to `overview` in `analysis-examples`

other areas touched:
- Added introductory text to landing page of the data services documentation **initial landing page**
- added remaining internal links to link out from data services documentation landing page

### CODE REVIEW

**Review by machow — CHANGES_REQUESTED** [review]
From pairing with @charlie-costanzo -- thanks, having these as tabbed examples look really nice! Can we try embedding the supplemental explanations that are in tables right now?

For when we have a table explaining what GTFS Schedule Fact Daily Feeds is above the metabase query example, let's move the explanation **into** the example. This will make it a bit easier for people to integrate the information into the example, and help us avoid duplicating a lot of pieces from the example.

image

File: docs/analytics_examples/01_warehouse_tutorial.md [hunk 1]
```diff
@@ -0,0 +1,689 @@
...
+(routes-agency-time)=
+### 1. Number of Routes for a Given Agency Over Time
+
+```{code-cell}
+:tags: [remove-cell]
+df_routes = query_sql("""
+SELECT
+    calitp_feed_name,
+    date,
+    count(*) AS count_feeds
```
Comment (evansiroky) [inline docs/analytics_examples/01_warehouse_tutorial.md:97]: This is counting the number of routes, right? Seems like this should be:

```suggestion
    count(*) AS count_routes
```

File: docs/analytics_examples/01_warehouse_tutorial.md [hunk 2]
```diff
@@ -0,0 +1,689 @@
...
+    >> select(_.calitp_feed_name, _.date, _.days_until_feed_end_date, _.feed_end_date)
+    >> filter(_.date == "2021-09-01", _.calitp_feed_name == "Unitrans (0)")
+)
+```
+```{glue:figure} siuba_feed_expires_output
+```
+````
+
+(max-number-stops)=
+### 5. Max Number of Stops a Trip Can Have, Per Agency
```
Comment (evansiroky) [inline docs/analytics_examples/01_warehouse_tutorial.md:552]: Query description suggestion:

```suggestion
### 5. Find the Trip With the Most Number of Stops, Per Agency
```

**Review by evansiroky — CHANGES_REQUESTED** [review]
See recommended changes, also please fix merge conflict.

**Comment by charlie-costanzo** [comment]
> See recommended changes, also please fix merge conflict.

Thanks for the review and suggested changed @evansiroky - I committed the changes and fixed the merge conflict.

**Review by evansiroky — APPROVED** [review]

**Comment by charlie-costanzo** [comment]
@machow now that the changes suggested by Evan are commited, should we merge this to close out existing docs PRs? 

The remaining piece - adding supplemental explanations - is now captured in it's own issue, #779

**Comment by machow** [comment]
@charlie-costanzo sounds good--thanks for the changes / catching the rest in an issue

**Pull request merged**